src/providers/bibtexformatter.ts

```diff
@@ -69,7 +69,9 @@ export class BibtexFormatter {
             right: leftright[1],
             trailingComma: config.get('bibtex-format.trailingComma') as boolean,
             sort: config.get('bibtex-format.sortby') as string[],
-            alignOnEqual: config.get('bibtex-format.align-equal.enabled') as boolean
+            alignOnEqual: config.get('bibtex-format.align-equal.enabled') as boolean,
+            sortFields: config.get('bibtex-fields.sort.enabled') as boolean,
+            fieldsOrder: config.get('bibtex-fields.order') as string[]
         }
```

Comment:
Here, we should add the `configuration` variable to log messages with `addLogMessage`.